core/themes/basis/theme-settings.php

```diff
@@ -70,3 +73,96 @@
     $form['footer'][$field] = color_get_color_element($form['theme']['#value'], $field, $form);
   }
 }
+
+$theme_name = $form['theme']['#value'];
+$theme_path = backdrop_get_path('theme', 'basis');
+require_once $theme_path . '/template.php';
+$install_version = config_get('system.core', 'install_version');
+
+$css_update_options = array(
+  'install_version' => t('Default'),
+  'all_updates' => t('All updates'),
+  'custom' => t('Specific version'),
+);
+
+$form['css_updates'] = array(
+  '#type' => 'details',
+  '#open' => FALSE,
+  '#summary' => t('CSS Updates'),
+  '#details' => t('There are occasionally updates available for Basis that may affect existing sites, changing this setting may cause unexpected changes to your theme. For more information and details, see the <a href="https://docs.backdropcms.org/documentation/layouts-and-templates" target="_blank">online documentation</a>.</br></br>Select which updates you would like to accept. '),
+  '#attributes' => array(
+    'class' => array('description'),
+  ),
+);
+
+// Load the saved configuration value.
+$form['css_updates']['css_update'] = array(
+  '#title' => t('CSS Update Options'),
+  '#type' => 'radios',
+  '#options' => $css_update_options,
+  '#default_value' => theme_get_setting('css_update', $theme_name),
+  '#config' => 'basis.settings',
+  'default' => array(
+    '#description' => t('Accept no updates introduced after installation.'),
+  ),
+  'all_updates' => array(
+    '#description' => t('Warning: Some future changes may effect your site.'),
```

Comment:
"effect" should be "affect"

Reply:
Nevermind. If the UI is removed, this is no longer relevant.

Reply:
I don't think it was decided to remove the UI. It would make it harder for contrib themes to set a minium version without the UI (since they can't run update hooks).

Reply:
See https://github.com/backdrop/backdrop-issues/issues/4782#issuecomment-2556007996, we're going to remove the UI from this PR and open a follow-up.

Reply:
We fixed the effect/affect problem in one place, but it seems I may have missed another.
  
Hoping to get to update this PR tonight. Not yet certain if that will be possible.

Reply:
I fixed this typo in the mean time anyway.